src/test/java/seedu/mindmymoney/command/CalculateInputCommandTest.java

```diff
@@ -23,66 +23,97 @@ public void setUp() {
     }
 
     /**
-     * Asserts if user is able to calculate expenditure per month.
+     * Asserts if user is able to calculate expenditure by date.
      */
     @Test
-    void calculateInputCommand_oneInput_expectCorrectOutput() throws MindMyMoneyException {
+    void calculateInputCommand_calculateByDate_expectCorrectOutput() throws MindMyMoneyException {
         ExpenditureList expenditureTestList = new ExpenditureList();
         CreditCardList creditCardTestList = new CreditCardList();
         IncomeList incomeList = new IncomeList();
         User user = new User(expenditureTestList, creditCardTestList, incomeList);
         String inputString = "/pm cash /c Personal /d Nike Shoes /a 300 /t 30/03/2022";
```

Comment:
Maybe can add a new line here to "break up" the code, so its not a continuous chunk of code
```suggestion
        
        String inputString = "/pm cash /c Personal /d Nike Shoes /a 300 /t 30/03/2022";
```